mmtrack/datasets/builder.py

```diff
@@ -56,10 +57,13 @@ def build_dataloader(dataset,
         DataLoader: A PyTorch dataloader.
     """
     rank, world_size = get_dist_info()
+    is_sotdataset = isinstance(dataset, BaseSOTDataset) or (
+        isinstance(dataset, ConcatDataset)
+        and isinstance(dataset.datasets[0], BaseSOTDataset))
```

Comment:
add comments for `isinstance(dataset, ConcatDataset) and isinstance(dataset.datasets[0], BaseSOTDataset)`